features/bootstrap/OrderManagingContext.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+use Behat\Behat\Context\Context;
+use Behat\Behat\Context\SnippetAcceptingContext;
+use Behat\Gherkin\Node\TableNode;
+use Behat\Behat\Tester\Exception\PendingException;
+use Broadway\CommandHandling\SimpleCommandBus;
+use Eaty\Application\Caterers;
+use Eaty\Application\Command;
+use Eaty\Application\Handler;
+use Eaty\Application\InMemoryCaterers;
+use Eaty\Application\Orders;
+use Eaty\Application\InMemoryOrders;
+
+
+class OrderManagingContext implements Context, SnippetAcceptingContext
+{
+    /**
+     * @var SimpleCommandBus
+     **/
+    private $commandBus;
+
+    /**
+     * @var string
+     **/
+    private $catererWithStartedOrderName;
+
+    /**
+     * @var Orders
+     */
+    private $orders;
+
+    /**
+     * @var Caterers
+     */
+    private $caterers;
+
+    public function __construct()
+    {
+        $this->commandBus = new SimpleCommandBus();
+        $this->orders = new InMemoryOrders();
+        $this->caterers = new InMemoryCaterers();
+
+        $this->commandBus->subscribe(
+            new Handler\CreateNewCaterer($this->caterers)
+        );
+        $this->commandBus->subscribe(
+            new Handler\StartNewOrderForCaterer($this->orders, $this->caterers)
+        );
+    }
+
+    /**
+     * @Given there are following caterers:
+     */
+    public function thereAreFollowingCaterers(TableNode $caterersParameters)
+    {
+        foreach ($caterersParameters as $catererParameters) {
+            $command = new Command\CreateNewCaterer($catererParameters['name']);
+            $this->commandBus->dispatch($command);
+        }
+    }
+
+    /**
+     * @When I start order for :catererName caterer
```

Comment:
I think we have to include information about who started an order, here UserId which could be random string would be enough for now also we need to distinguish orders so each Order should contain OrderId (uuid).